config/routes.rb

```diff
@@ -324,6 +324,7 @@
   namespace :integrations, path: '/integrations' do
     post '/notify/callback' => 'notify#callback'
     get '/feature-flags' => 'feature_flags#index'
+    get '/performance-dashboard' => 'performance_dashboard#dashboard', as: :performance
```

Comment:
Do we want `dashboard` in the URL?

Reply:
Yeah, I think that makes the purpose clearer.